### PR TITLE
Orbit Animation Example

### PR DESCRIPTION
**Pull request by NigelBreslaw**

### CODE REVIEW

**Comment by ogoffart** [comment]
[Slintpad link](https://slintpad.com/?load_url=https://raw.githubusercontent.com/slint-ui/slint/3cf9e75ed33dd9dce98a5d17f32985c67a5851cf/examples/orbit-animation/demo.slint)

Looks nice. 
Is there something to see in Demo 4 ? it's all black to me.

**Comment by tronical** [comment]
https://github.com/user-attachments/assets/ab096330-8f56-448b-8a23-0b2951928f08

File: examples/orbit-animation/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+![Orbit Animation Screenshot](https://github.com/user-attachments/assets/a06bd1b3-fbb0-4b90-91c7-0a3fc181aae0)
+
+# Orbit Animation Demo
+
+Demonstrate how to fake simple orbit animations
```
Comment (tronical) [inline examples/orbit-animation/README.md:6]: Suggestions :)

```suggestion
Demonstrates how to fake orbit simple animations using Slint's trigonometric math functions.
```

File: examples/orbit-animation/images/slint-logo.svg
Comment (ogoffart) [inline examples/orbit-animation/images/slint-logo.svg:1]: Would it make sense to use an existing logo from `../../logos/`?
Then when we update the branding in some years, the demo will automatically be updated (with the risk that it is wrongly updated)
This is how i'd do it, i don't know if it is better to duplicate the logo or reference an existing one.

**Review by ogoffart — APPROVED** [review]
Demo 4 doesn't work for me with firefox because somehow it doesn't load some SVGs
But this is not a bug in the demo.

File: examples/orbit-animation/images/slint-logo.svg
Comment (NigelBreslaw) [reply examples/orbit-animation/images/slint-logo.svg:1]: I re-used the one from logos. If there are demos that people may copy paste into their project, then being self contained and having a copy might make sense. But this isn't one of those demos.

**Pull request merged**